### PR TITLE
feat:Create a new Angular 11 app with angular CLI[295]

### PR DESCRIPTION
**Pull request by AbdoulayeNDAO121**
Creating pull request for Initialise Client FrontEnd in Repository step 1

### CODE REVIEW

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({
+      spec: {
+        displayStacktrace: StacktraceOption.PRETTY
+      }
+    }));
+  }
+};
```
Comment (morlombakop) [inline e2e/protractor.conf.js:37]: Please add a new line at the end of this file

**Review by morlombakop — COMMENTED** [review]
@AbdoulayeNDAO121  , it looks good
Please fix the comments and we ready to merge it

File: e2e/protractor.conf.js (hunk 1 above)
Comment (AbdoulayeNDAO121) [reply e2e/protractor.conf.js:37]: Ah i did it in the others branches,
Ok i 'm fixing it right now

Comment (AbdoulayeNDAO121) [reply e2e/protractor.conf.js]: It's fixed !

**Review by morlombakop — APPROVED** [review]

**Pull request merged**